docs/docs/config/kscan.md

```diff
@@ -152,6 +152,43 @@ The output pins (e.g. columns for `col2row`) should have the flag `GPIO_ACTIVE_H
     };
 ```
 
+## Charliplex Driver
+
+Keyboard scan driver where keys are arranged on a matrix with each GPIO used as both input and output.
+
+- With `interrupt-gpios` unset, this allows n pins to drive n\*(n-1) keys.
+- With `interrupt-gpios` set, n pins will drive (n-1)\*(n-2) keys, but provide much improved power handling.
+
+Definition file: [zmk/app/drivers/kscan/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/drivers/kscan/Kconfig)
```

Comment:
FYI, all of the links in this file are wrong, they haven't been updated to point to the `/module` directory.

I leave this one in the same state deliberately incase the fix is done with a find/replace which may then break this one.

Reply:
```suggestion
Definition file: [zmk/app/module/drivers/kscan/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/module/drivers/kscan/Kconfig)
```